pkgs/tools/virtualization/marathonctl/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "marathonctl-${version}";
```

Comment:
The name should be `marathonctl-unstable-${version}` since it is built from a checkout that isn't formally released.